### ISSUE TITLE
IE11 breaks without polyfill because of Object.assign

### ISSUE THREAD

**Issue by elidupuis** [machty/ember-concurrency#60]
Just updated to 0.7.7 in order to remove the Babel polyfill but I noticed that IE11 breaks because it doesn't support the [few cases of `Object.assign`](https://github.com/machty/ember-concurrency/search?utf8=%E2%9C%93&q=%22Object.assign%22&type=Code) in ember-concurrency.

Obviously everything works fine with `includePolyfill: true` again.

**Comment by machty** [comment]
Can you try running with

```
    "ember-concurrency": "git://github.com/machty/ember-concurrency#a327c30b0064b430151be97c3d55929c8055dfe8",
```

in your package.json and let me know if the tests pass?

**Comment by elidupuis** [comment]
@machty Works for me!

**Comment by machty** [comment]
Nice, fixed on ember-concurrency#0.7.8
